packages/utilities/fast-web-utilities/src/key-codes.ts

```diff
@@ -143,3 +143,23 @@ export const keyCodeTilde: number = 192;
 export const keyCodeWindowsLeft: number = 91;
 export const keyCodeWindowsOpera: number = 219; // Opera
 export const keyCodeWindowsRight: number = 92;
+
+/**
+ * String values for use with KeyboardEvent.key
+ */
+export const keyArrowDown = "ArrowDown";
+export const keyArrowLeft = "ArrowLeft";
+export const keyArrowRight = "ArrowRight";
+export const keyArrowUp = "ArrowUp";
+
+export const keyEnter = "Enter";
+export const keyEscape = "Escape";
+export const keySpace = " ";
+export const keyTab = "Tab";
+
+export enum ArrowKeys {
```

Comment:
Can this be a type instead of an enum?

Reply:
This is used as a collection of values so a type won't work in its place.

Reply:
We use types for collections of values in a bunch of places:
ie. `export type VerticalPosition = "top" | "bottom" | "unset";`
Apparently there are perf benefits.

Reply:
I updated the `ArrowKeys` enum to now be a const with a combined type.